<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EnterCaptcha</class>
 <widget class="QDialog" name="EnterCaptcha">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>246</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>246</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>246</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Enter captcha</string>
  </property>
  <widget class="QDialogButtonBox" name="submitCaptcha">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>190</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
   <property name="centerButtons">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLineEdit" name="captchaText">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>150</y>
     <width>331</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="captchaImage">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>29</y>
     <width>331</width>
     <height>111</height>
    </rect>
   </property>
   <property name="text">
    <string>Loading captcha...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="errorMessage">
   <property name="geometry">
    <rect>
     <x>33</x>
     <y>170</y>
     <width>331</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: red;</string>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>submitCaptcha</sender>
   <signal>rejected()</signal>
   <receiver>EnterCaptcha</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
